### PR TITLE
cohort 4: schedule links to details in ACTIVITIES.md

### LINKED ISSUE #364: Schedule should have specific links with specific times
Ex. instead of just linking to the brainstorming session, speficically link to the yes and session, and then the post it note section

### PR DESCRIPTION
**Pull request by maxwofford**
This closes #364

### CODE REVIEW

**Comment by maxwofford** [comment]
@zachlatta Would you please review and merge this?

File: cohort_4/playbook/SCHEDULE.md [hunk 1]
```diff
@@ -67,26 +67,36 @@ partners.
...
+| 2:00 PM  | [Brainstorming](ACTIVITIES.md#brainstorming)                               |
+| 2:00 PM  | [Forming groups](ACTIVITIES.md#forming-groups)                             |
+| 2:05 PM  | [Yes-and sessions](ACTIVITIES.md#yes-and-sessions)                         |
+| 2:10 PM  | [Running a brainstorm session](ACTIVITIES.md#running-a-brainstorm-session) |
+| 2:15 PM  | [After a brainstorm](ACTIVITIES.md#after-a-brainstorm)                     |
+| 2:20 PM  | [Project time](ACTIVITIES.md#freeform-projects)                            |
+|          | [Scoping and concept](ACTIVITIES.md#scoping-and-concept)                   |
+|          | [Engagement](ACTIVITIES.md#engagement)                                     |
+|          | [Facilitating](ACTIVITIES.md#facilitating)                                 |
+|          | [Answering questions](ACTIVITIES.md#answering-questions)                   |
```
Comment (zachlatta) [inline cohort_4/playbook/SCHEDULE.md]: Would it make sense to add a time for each section to say when each section should happen? Also - Markdown tables can only have one entry per line.

File: cohort_4/playbook/SCHEDULE.md [hunk 2]
```diff
@@ -96,34 +106,52 @@ partners.
...
+|          | [Engagement](ACTIVITIES.md#engagement)                       |
+|          | [Facilitating](ACTIVITIES.md#facilitating)                   |
+|          | [Answering questions](ACTIVITIES.md#answering-questions)     |
 | 12:30 PM | [Lunch](ACTIVITIES.md#lunch)                                 |
 | 1:30 PM  | Energizer                                                    |
 | 2:00 PM  | [Project time](ACTIVITIES.md#freeform-projects)              |
+|          | [Scoping and concept](ACTIVITIES.md#scoping-and-concept)     |
+|          | [Engagement](ACTIVITIES.md#engagement)                       |
+|          | [Facilitating](ACTIVITIES.md#facilitating)                   |
+|          | [Answering questions](ACTIVITIES.md#answering-questions)     |
```
Comment (zachlatta) [inline cohort_4/playbook/SCHEDULE.md]: Same thing here. Does it make sense to link to each individual section from here? If they just go to the "Scoping and concept" section, they should be able to see all of these individual sections.

File: cohort_4/playbook/SCHEDULE.md (hunk 1 above)
Comment (maxwofford) [inline cohort_4/playbook/SCHEDULE.md]: How can I schedule time for Facilitating vs Answering questions?

Comment (zachlatta) [inline cohort_4/playbook/SCHEDULE.md]: Would it make sense to have a time scheduled for that? Perhaps just link to
"Project time" and in the project time guidelines have an index at the top?

Ex.

---

# Project Time
- [Scoping and concept](ACTIVITIES.md#scoping-and-concept)
- [Engagement](ACTIVITIES.md#engagement)
- [Facilitating](ACTIVITIES.md#facilitating)
- [Answering questions](ACTIVITIES.md#answering-questions)

On Fri, Aug 7, 2015 at 11:50 AM, Max Wofford notifications@github.com
wrote:

> In cohort_4/playbook/SCHEDULE.md
> https://github.com/hackedu/hack-camp/pull/366#discussion_r36549345:
> 
> > +| 9:50 AM  | [Energizer 2](ACTIVITIES.md#i-love-my-neighbor-who)                        |
> > +| 10:00 AM | [Breakfast](ACTIVITIES.md#breakfast)                                       |
> > +| 10:30 AM | [Introduction talk](ACTIVITIES.md#introduction-talk)                       |
> > +| 10:40 AM | [Portfolio workshop](ACTIVITIES.md#portfolio)                              |
> > +| 12:30 PM | [Lunch](ACTIVITIES.md#lunch)                                               |
> > +| 9:50 AM  | [Energizer 3](ACTIVITIES.md#evolution-rock-paper-scissors)                 |
> > +| 2:00 PM  | [Brainstorming](ACTIVITIES.md#brainstorming)                               |
> > +| 2:00 PM  | [Forming groups](ACTIVITIES.md#forming-groups)                             |
> > +| 2:05 PM  | [Yes-and sessions](ACTIVITIES.md#yes-and-sessions)                         |
> > +| 2:10 PM  | [Running a brainstorm session](ACTIVITIES.md#running-a-brainstorm-session) |
> > +| 2:15 PM  | [After a brainstorm](ACTIVITIES.md#after-a-brainstorm)                     |
> > +| 2:20 PM  | [Project time](ACTIVITIES.md#freeform-projects)                            |
> > +|          | [Scoping and concept](ACTIVITIES.md#scoping-and-concept)                   |
> > +|          | [Engagement](ACTIVITIES.md#engagement)                                     |
> > +|          | [Facilitating](ACTIVITIES.md#facilitating)                                 |
> > +|          | [Answering questions](ACTIVITIES.md#answering-questions)                   |
> 
> How can I schedule time for Facilitating vs Answering questions?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/pull/366/files#r36549345.

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by maxwofford** [comment]
These commits must be redone with rebased changes.

**Pull request closed without merging**